src/main/java/com/napier/sem/App.java

```diff
@@ -85,14 +85,23 @@ public static void main(String[] args) {
         // Produce a report on all capital cities in a region organised by largest population to smallest
         // ArrayList<Record> capitalCities = dao.allCapitalCitiesIn(REGION, "Caribbean");
 
+
+        // Use-Cases 6.1-6.3
+        // 6.1 - Produce a report on the top N capital cities in the world organised by largest population to smallest
+        // ArrayList<Record> capitalCities = dao.topNCapitalCitiesIn(WORLD, "",5);
+        // 6.2 - Produce a report on the top N capital cities in the world organised by largest population to smallest
+        // ArrayList<Record> capitalCities = dao.topNCapitalCitiesIn(CONTINENT, "Asia", 5);
+        // 6.3 - Produce a report on the top N capital cities in the world organised by largest population to smallest
```

Comment:
these comments need to be updated to reflect the actual use cases, i.e. top N capital cities in a continent organised by largest population to smallest